.github/workflows/ci.yml

```diff
@@ -127,9 +170,7 @@ jobs:
       - name: Install Test dependencies
         timeout-minutes: 2
         run: |
-          dnf install -y python3 wget \
-            $(grep '^Dependencies(dnf): ' dotnet-regular-tests/README.md | cut -d: -f2-) \
-            --skip-broken
+          dnf install -y python3 wget $(grep '^Dependencies(dnf): ' README.md | cut -d: -f2-) --skip-broken
```

Comment:
It looks like the removal of `dotnet-regular-tests/` leads to missing certain test dependencies, like `nm`.